python/ray/dashboard/modules/train/train_head.py

```diff
@@ -332,6 +327,14 @@ async def get_train_runs(self, req: Request) -> Response:
             content_type="application/json",
         )
 
+    async def _get_actor_infos(self, actor_ids: List[str]):
+        actor_ids_qs_str = ",".join(actor_ids)
+        url = f"http://{self.http_host}:{self.http_port}/logical/actors?ids={actor_ids_qs_str}&nocache=1"
+        async with self._http_session.get(url) as resp:
+            assert resp.status == 200, f"Failed to get actor info: {resp.status}"
```

Comment:
lets' not use assert, but returns the 500 to the end user

Reply:
assert is used to catch program logical errors. Lets raise a different error instead.

Reply:
Updated. Use `resp.raise_for_status()` instead.